geniza/corpus/management/commands/import_data.py

```diff
@@ -298,7 +299,8 @@ def add_document_language(self, doc, row):
 
     def import_document(self, row):
         """Import a single document given a row from a PGP spreadsheet"""
-        if ";" in row.type:
+        # skip any row with multiple types or flagged for demerge
+        if ";" in row.type or "DISAGGREGATE" in row.notes:
```

Comment:
"DISAGGREGATE" occurs in the "Technical notes (optional)" column for 8918 rows, whereas in "Notes2 (optional) it occurs in 1925 rows. Should we consider both? Has that already been discussed?

Reply:
@kmcelwee the last time I looked, there were no more DISAGGREGATE entries in the technical notes (and I just confirmed via colab). I think they've been handled, and thus were removed. Otherwise, we should definitely check there also! 

Can you easily check if the records we're skipping match up with what's in the demerge spreadsheet? Or would the import reporting flag it if they weren't ?

Reply:
I'm going to go ahead and merge — we can tweak the disaggregate skipping logic if needed